LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2016 Caleb Meredith
```

Comment:
[*cough cough cough*](https://github.com/ardite/ardite-core/graphs/contributors) 😊

Considering this code is like 90% mine I think this is fair.